.github/workflows/auto-merge-bot.yml

```diff
@@ -0,0 +1,21 @@
+on: [pull_request]
+
+jobs:
+  hello_world_job:
+    runs-on: ubuntu-latest
+    name: EIP Auto-Merge Bot
+    steps:
+      # To use this repository's private action,
+      # you must check out the repository
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js Enviornment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
```

Comment:
node 12 is required (only supported option)

Reply:
Hmm, node 14 isn't supported?  14 is current active version of NodeJS, so I'm surprised that only 12 is supported.  😢 
https://nodejs.org/en/about/releases/

Reply:
Unfortunately yes, if you try anything but node 12 it tells you to use docker, and I doubt that's worth the hassle.

Although idk if you setup the node as 14 it may just use node 14.

Either way typescript us using es6 so should be too much of an issue

https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-an-action-metadata-file